### DISCUSSION TITLE
我皮肤一直改不动

### DISCUSSION THREAD

**Discussion by chenghuang0801** [Meloong-Git/PCL#3153] [提问] [answered]
就刚买到手的时候弄了个黑人小哥皮肤（自带的），之后不管我用pcl还是官服渠道改皮肤我账号的皮肤显示还是显示是黑人小哥，就算改了，过段时间也会莫名其妙变回去，甚至就算我在官方那边改成了史蒂夫，进去后也变成了黑人小哥，为啥啊服了

**Comment by burningtnt** [comment]
你是微软账户吗？

**Reply by chenghuang0801** [reply]
是，官方在线版，不玩离线

**Reply by burningtnt** [reply, marked as answer]
在 PCL2 内删除原来的账户，重新登录一下

**Reply by chenghuang0801** [reply]
删除后好像有效果，感谢
